### ISSUE TITLE
Bug: .92 version - Chef Inv Full

### ISSUE THREAD

**Issue by NerdyMindNYC** [TangoTek/TekTopia-Community#915]
**Describe the bug**
Too much beetroot soup, nowhere to put it.

**To Reproduce**
Ver update, chefs now hold 15 soups.  at 13-14 inventory is full so they never drop them off.  As a result, they do nothing else because they can't pickup items to craft any other food.

**Screenshots**
Our chef has been in the library all day because their inventory is full.
![2019-08-13_14 49 02](https://user-images.githubusercontent.com/51106650/62969178-b2b16080-bdda-11e9-8a8a-b3368224d89e.png)
![2019-08-13_14 52 14](https://user-images.githubusercontent.com/51106650/62969179-b2b16080-bdda-11e9-8dc3-62b4736181b9.png)
![2019-08-13_14 54 39](https://user-images.githubusercontent.com/51106650/62969180-b2b16080-bdda-11e9-8c20-a9550cf03c6b.png)

** Server/Location **
What server were you playing on? Main(Survival) or Dev (Creative)? Main
What are the world coordinates of your town hall marker? -2850 63 11180

**Additional context**
Half would be okay, maybe 7 or 8

**Comment by mrhyde1st** [comment]
Indeed, all villagers probably need a default check built into their classes - If inventory = full, return goods...
